GG/src/SDL/SDLGUI.cpp

```diff
@@ -512,6 +512,19 @@ bool SDLGUI::Fullscreen() const
 bool SDLGUI::FakeModeChange() const
 { return m_fake_mode_change; }
 
+std::string SDLGUI::ClipboardText() const {
+    if (SDL_HasClipboardText()) {
+        char* text = SDL_GetClipboardText();
+        if (text) {
+            std::string result(text);
+            SDL_free(text);
+            return result;
+        }
+    }
+
+    return std::string();
```

Comment:
Literally no reason.  `std::string()` seemed conceptually simpler to me because of my mindset at the time.  I can change it if you have a strong (or really any) preference.